### DISCUSSION TITLE
台灣周休三日提案的逆向思考

### DISCUSSION THREAD

**Discussion by noratsaiiii** [1101-GS4719-C/Python-Community-GS4719-C#1442] [General]
曾經想像過週末仍得上半天課的時代嗎？事實上，直到1998年，台灣才開始實行「隔週」週休二日政策，直到2001年才全面實施每周週休二日。然而，近日，台灣網友Lucas在公共政策網路參與平台上發起了一項連署活動，希望讓台灣成為亞洲第一個實行週休三日的國家。

Lucas指出，根據2021年的統計數據顯示，台灣的平均年總工時雖已降至2000小時，創下歷年最低紀錄，但仍然是全球第四高的國家。在歐美地區，一些企業已經開始實施週休三日的實驗，讓員工在工作壓力減輕的同時提升工作意願。這項提案迅速引起了大眾的關注，截至目前已經有5736人參加了連署。提案通過之後，政府將於6月26日星期一正式回應周休三日的政策建議，這也引發許多上班族的矚目與期待。

然而，我發現也有人對此提出了反對意見。他們擔心這將導致失業率上升，因為外籍勞工可能會更迅速地填補空缺。另外，也有人質疑週休三日是否能確保工作效率的提升，並對經濟產生負面影響，特別是股市休息時。

整體而論，個人認為週休三日的政策是弊大於利。周休三日有助於提升勞工的生活品質和幸福感。多一天的休息時間可以減輕工作壓力，提高生活質量。然而，我覺得週休三日會對經濟產生負面影響不容小覷。減少工作時間會導致生產力下降，影響企業的運作效率，進而對經濟造成損害。這可能導致外來勞工取代本地勞動力，造成失業率上升的風險。此外，我認為在全球競爭激烈的環境中，台灣企業需要更高的工作效率才能保持競爭力。

**Comment by tonyYZL** [comment]
現在的人對生活品質越來越講究，有更多的假日看似是一件好事，但是相對的工時減少，收入也會因此變少。我覺得台灣現在的薪資跟物價相比已經算低了，如果收入又減少，即使有更多假日，我們也無法負擔更多額外的開銷。（不過對躺平族來說就是賺）

**Comment by sjh501** [comment]
文中有提到提倡周休三日的一個原因是工時過長，但我認為周休三日並非是解決台灣高工時問題的方法。就我的父母和周遭有在上班的人來說加班是家常便飯，尤其從事責任制工作，周休三日會變相是將一天的工作量分攤到另外四天，一周的工時不會改變，但單日的工時增長。如此一來加班費對於公司財政是一大負擔，對於工作的受雇者而言也未必比較輕鬆。因此我覺得以台灣的工作環境和生態而言，周休三日只是治標不治本的方法，且可能所造成負面影響不容小覷。

**Comment by Alexgns** [comment]
台灣的科技產業確實需要高工時的工作，要如何在台灣的經濟發展以及生活兩者間平衡極為重要，因此不應貿然做出決定，否則對經濟可能影響甚大

**Comment by ALAN30359704** [comment]
我認為周休三日會讓台灣的產業造成極大的影響原因是我們台灣人已經習慣週休二日的工作模式若突然改成周休三日反而會讓人產生怠惰的行為，而且會將原本五天的工作量壓縮成十天的工作量那單位時間內所需要完成的工作量也會變多我認為這不是件增加生活品質的好辦法！

**Comment by zephyrxin01** [comment]
我覺得以台灣的產業結構來說週休三日可能帶來的衝擊可能不小，也因此政府更應該去審慎思考並且評估這個政策的必要性以及對勞工實際帶來的福祉到底高不高，否則貿然推行可能反而造成反效果。

**Comment by youtinglin** [comment]
我個人認為若改成週休三日， 一定要確保勞工相關法規有清楚的規定，並且保障勞工的基本權利，避免因週休三日，導致不公平的加班，或者是沒有薪水但仍需超時工作，反而造成員工被剝削的更嚴重。而且在台灣可能會造成就算有週休三日，但仍會因有工作或職務在身，也無法確實好好放假，且造成經濟上有一定的影響，這件事的根本應該不是要週休三日，而是要讓每間公司都能讓員工能有一定的福利和假，也就能提高工作效率和生活品質。

**Comment by Elaine2899** [comment]
我認為周休三日能提高平日的工作效率，也增加人們的幸福感。但這樣的想法終究是理想，因為台灣的環境或許不適合我們改成周休三日，一方面是法規跟不上時代進而無法保護勞工和資方，另一方面是台灣整體的社會風氣，致使我們沒辦法乖乖地在上班時間把該做的事情做完，而資方也不會這麼輕易就放人回家。
